File: Strings/AnagramString.java
package Strings;

import java.util.Arrays;

public class AnagramString {
    public static String SortString(String s){
        char ch[]=s.toCharArray();
        Arrays.sort(ch);
        return new String(ch);
    }
public static boolean checkAnagramString(String str1, String str2 ){
    if(str1.length()!=str2.length()){
        return false;
    }
    str1=SortString(str1);
    str2=SortString(str2);
    for(int i=0;i<str1.length();i++){
        if(str1.charAt(i)!=str2.charAt(i)){
            return false;
        }
    }
        return true;
}
public static void main(String[] args) {
    String str1="CAT";
    String str2="TAC";
    if(checkAnagramString(str1, str2)){
        System.out.println("The strings are in anagram.");
    }else{
        System.out.println("The strings  are not  in anagram.");
    }
  }
}

File: Problem Solving Python/Strings/Anagram_String.py
def AnagramString()->bool:
    str1=sorted(input("Enter your first string: "))
    str2=sorted(input("Enter your second string: "))
    n=len(str1)
    m=len(str2)

    if(n!=m):
        return False
    for i in range(n):
        if(str1[i]!=str2[i]):
            return False
    return True

if __name__=='__main__':
    if(AnagramString()):
        print("The two Strings are Anagram string")
    else:
        print("The two Strings are not Anagram string")
